qiita_pet/support_files/doc/source/faq.rst

```diff
@@ -244,6 +244,89 @@ etc; for more information check
 `this 3 minute read about Markdown <https://guides.github.com/features/mastering-markdown/>`__.
 
 
+I want to transfer a lot of files to Qiita, is there an easy way?
+-----------------------------------------------------------------
+
+Yes! This is available in the "Upload Files" section of each study by accessing the tab
+"Upload via Remote Server (ADVANCED)".
+
+We currently suggest using scp, note that your server where you store your files should allow
+connecting to it via scp - in other words, this only works when moving files from a server to Qiita.
+
+Now, the way it works is that you need to create a new secure key to that server, imagine
+that you are making a copy of your storage-shed key, then you share that key to Qiita (you will give
+access to Qiita to run a single copy command in your server), Qiita uses that key and securely
+destroy it.
+
+To take advantage of this feature you need to:
+
+#. Prepare all the files you want to transfer in a single folder (or you can do this on
```

Comment:
can do this on multiple folders -> can transfer multiple times from multiple folders into one study